packages/api/models/granules.js

```diff
@@ -315,6 +315,7 @@ class Granule extends Manager {
     executionUrl,
     executionDescription = {}
   ) {
+    if (!granule.granuleId) throw new Error(`Could not create granule record, invalid granuleId: ${granule.granuleId}`);
```

Comment:
if `granule.granuleId` is falsy or let's assume undefined, then the log message here will always say `invalid granuleId: undefined`, I think?

Reply:
Yeah. I went with falsy because `N/A` or `""` or `false` or whatever are also clearly not valid granule IDs. I don't know how better to clarify that it is not a value we expect.

Reply:
Yeah, I think it's fine